Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:focal as build
+LABEL maintainer Elmo Todurov <elmo.todurov@eesti.ee>
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=utc apt-get install --yes --no-install-recommends build-essential libboost-dev libboost-program-options-dev libboost-regex-dev libboost-filesystem-dev libboost-system-dev libmariadbclient-dev scons libcurl4-openssl-dev git ca-certificates
+ARG UID=1000
+RUN useradd -m -u $UID sneezy
+USER sneezy
+ARG BRANCH="master"
+# COPY . /home/sneezy/sneezymud
+WORKDIR /home/sneezy
+ARG FORCE_REBUILD=1
+RUN echo Building from branch: $BRANCH; git clone --depth 1 --shallow-submodules --recurse-submodules --single-branch --branch $BRANCH --no-tags https://github.com/sneezymud/sneezymud
```

Comment:
So I'd really love to remove this but it didn't work locally to just copy the code into the container as on line 9

Reply:
Huh, why? It's good practice to build inside the container, as it'll stabilize your build environment (Ubuntu updates in the pipeline's worker agents won't cause build to break because Boost updated a version, or such.

Reply:
Yeah exactly. Ok so since there might be some additional bits to work out I'm going to leave it this was and get back to it another time.